website/docs/getting-started/getting-started-guide.md

```diff
@@ -0,0 +1,185 @@
+
+![](https://i.imgur.com/jexwOeu.png)
+Copy the token and then Open a terminal and type in `ksctl cred`
+
+```bash
+ksctl cred 
+
+1> AWS (EKS)
+2> Azure (AKS)
+3> Civo (K3s)
+
+```
+
+Then, authenticate it with the respected cloud provider. In this case, we are using Civo, so we will type 3 
+paste your API key when prompted into the command line and hit enter.
+
+You should then see the following success message:
+
+:::success success
+[INFO] 💾 configuration
+
+Login Success
+:::
+Now, ksctl is connected to your civo account. We can now move ahead and create a cluster.
+
+## Create a Cluster
+
+Ensure that you are connected the correct cloud provider. After adding your cloud cred to ksctl, 
+you can use the following CMD to create a cluster.
+
+```bash
+ksctl create-cluster --help
+It is used to create cluster with the given name from user. For example:
+
+ksctl create-cluster ["azure", "gcp", "aws", "local"]
+
+Usage:
+  ksctl create-cluster [command]
+
+Aliases:
+  create-cluster, create
+
+Available Commands:
+  aws         Use to create a EKS cluster in AWS
+  azure       Use to create a AKS cluster in Azure
+  civo        Use to create a CIVO k3s cluster
+  ha-azure    Use to create a HA k3s cluster in Azure
+  ha-civo     Use to create a HA CIVO k3s cluster
+  local       Use to create a LOCAL cluster in Docker
+
+Flags:
+  -h, --help   help for create-cluster
+
+Use "ksctl create-cluster [command] --help" for more information about a command.
+```
+
+This will have some options specific to the cloud provider. Below, we'll take a look at how you can create clusters in civo.
+
+Apart from using the `ksctl create civo` command, we can also insert some flags. 
+Some flags which will be common across every provider are `Cluster name`, `Node size`, `Region`, and `Number of nodes`.
+
+```bash
+ksctl create-cluster civo --help
+It is used to create cluster with the given name from user. For example:
+
+ksctl create-cluster civo <arguments to civo cloud provider>
+
+Usage:
+  ksctl create-cluster civo [flags]
+
+Flags:
+  -a, --apps string       PreInstalled Apps with comma seperated string
+  -c, --cni string        CNI Plugin to be installed
+  -h, --help              help for civo
+  -n, --name string       Cluster name
+  -s, --nodeSize string   Node size (default "g4s.kube.xsmall")
+  -N, --nodes int         Number of Nodes (default 1)
+  -r, --region string     Region
+  -v, --verbose           for verbose output (default true)
+```
+
+Here, we will create a medium-sized cluster, named demo-cluster which will have pre-install ArgoCD from the civo marketplace.
+
+```bash
+ksctl create-cluster civo --name demo --nodeSize g4s.kube.medium --nodes 3 --region LON1 --apps argo-cd
+```
+
+After you create your cluster, you will get a prompt asking if you want to PRINT your kubeconfig.
+
+> Note: It will only print the kubeconfig, not save it to your local `.kube` folder.
+
+![](https://i.imgur.com/vJunAfl.png)
```

Comment:
I think we should have this section as generic provider rather than having only for civo

or if we want to have it specific we need to add it to their respective provider page and then link it here

Reply:
@dipankardas011 I would prefer to have a generic provider section so that it can cover all the cloud provider rather than using any single cloud one